crates/bevy_gizmos/src/primitives/helpers.rs

```diff
@@ -0,0 +1,94 @@
+use std::f32::consts::TAU;
+
+use bevy_math::{Mat2, Quat, Vec2, Vec3};
+use bevy_render::color::Color;
+
+use crate::prelude::{GizmoConfigGroup, Gizmos};
+
+// helpers - affine transform
+
+pub fn rotate_then_translate_2d(angle: f32, position: Vec2) -> impl Fn(Vec2) -> Vec2 {
```

Comment:
I would really like docs for these public methods.

Reply:
Good catch 🎯 They are exposed by the `pub(crate)` helper module but I guess I should've made them `pub(crate)` themselves as well. That's easier to read for the bevy devs which just jump there and are confused why they can't access it elsewhere.

I did some documentation there as well anyways